test/python/opflow/test_gradients.py

```diff
@@ -951,6 +955,49 @@ def test_vqe2(self):
         result = vqe.compute_minimum_eigenvalue(operator=h2_hamiltonian)
         np.testing.assert_almost_equal(result.optimal_value, h2_energy, decimal=0)
 
+
+    W1 = np.array([
+        [0, 1, 0, 1],
+        [1, 0, 1, 0],
+        [0, 1, 0, 1],
+        [1, 0, 1, 0]
+    ])
+    P1 = 1
+    M1 = (I ^ I ^ I ^ X) + (I ^ I ^ X ^ I) + (I ^ X ^ I ^ I) + (X ^ I ^ I ^ I)
+    S1 = {'0101', '1010'}
+
+    CUSTOM_SUPERPOSITION = [1 / np.sqrt(15)] * 15 + [0]
+
+    @idata([
+        [W1, P1, M1, S1]
+    ])
+    @unpack
```

Comment:
I don't think this wrapping in decorators is needed since we only check a single case, could you remove that?